### ISSUE TITLE
Chest invisibles and Bug

### ISSUE THREAD

**Issue by Antoniovr07** [PrinceOfAmber/Storage-Network#293]
Minecraft version & Mod Version:1.16.1
Single player or Server:Single player
Describe problem:I don't know very well how the mod works but the chests are invisible and sometimes in the chat I get "OpenGL error: 1283 (Stack Overflow)".

**Comment by lilmayu** [comment]
Yup, all entities and tile-entities are invisible upon hovering search box in storage block. I can provide video, if needed.

**Comment by Ollek91** [comment]
Same problem.
When you click on the storage request tabel, your hand disappears and everything you disassemble will be invisible.

**Comment by shanekremer** [comment]
Same issue. This mod is completely unusable for now.

**Comment by Lothrazar** [comment]
I cant replicate this for the life of me.

Are you using optifine?

What version of forge are you using?

Im using the newest possible 1.16 forge here, no optifine, no other mods at all. 

Recorded a video to be sure.  Did the two steps listed of opening the table and hovering over the search bar.

If anyone has either a crash log, or a video or list of other mods i could look into it. but so far im 80% sure its a different mod causing this?

VIDEO  
![293](https://user-images.githubusercontent.com/621529/90721588-ecc70500-e26d-11ea-953f-02b1594dcef9.gif)

**Comment by krazzykid2006** [comment]
Mine aren't going invisible but it is crashing the instant I try to open the interface.
1.16.1 
Latest forge, 1.16.1-32.0.108
SimpleStorageNetwork-1.16.1-1.0.0.jar
Simple storage is the only mod I have installed, so definitely not another mod causing it.
Not using optifine either.

I don't at all see how it is working for you in that video above when it has not once worked for me with zero other mods.
I've tried many forge versions as well and same exact result with every version I have tried.
[crash-2020-08-20_22.57.57-client.txt](https://github.com/Lothrazar/Storage-Network/files/5106731/crash-2020-08-20_22.57.57-client.txt)

Edit
I've tried both with and without other mods, same result.
This install has had nothing except forge and simple storage though. Instant crash when trying to open interface.
Same issue with only this mod and JEI installed too.
